tests/test_catalyst/test_openinference_langchain.py

```
@@ -0,0 +1,562 @@
+import os
+# import sys
+# sys.path.append('/Users/ragaai_user/work/ragaai-catalyst/')
+import time
```

Ruff (0.8.2)

`time` imported but unused

Remove unused import: `time`

(F401)